pkg/kubernetes/client/client.go

```diff
@@ -11,7 +11,7 @@ import (
 
 // OVNKMCPServerClientSet is a client set for the OVN Kubernetes MCP server.
 type OVNKMCPServerClientSet struct {
-	clientSet                   kubernetes.Interface
+	ClientSet                   kubernetes.Interface
```

Comment:
As I mentioned earlier, the k8s client is not exported because the other layers can simply use the `ExecPod` tool.